scripts/posteffects/posteffect-ssao.js

```diff
@@ -377,23 +377,15 @@ function SSAOEffect(graphicsDevice, ssaoScript) {
     this.brightness = 0;
     this.samples = 20;
     this.downscale = 1.0;
-
-    this.resize(null);
 }
 
 SSAOEffect.prototype = Object.create(pc.PostEffect.prototype);
 SSAOEffect.prototype.constructor = SSAOEffect;
 
-SSAOEffect.prototype.resize = function (target) {
+SSAOEffect.prototype._resize = function (target) {
 
-    var width, height;
-    if (target === null) {
-        width = Math.ceil(this.device.width / this.device.maxPixelRatio / this.downscale);
-        height = Math.ceil(this.device.height / this.device.maxPixelRatio / this.downscale);
-    } else {
-        width = Math.ceil(target.colorBuffer.width / this.device.maxPixelRatio / this.downscale);
-        height = Math.ceil(target.colorBuffer.height / this.device.maxPixelRatio / this.downscale);
-    }
+    var width = Math.ceil(target.colorBuffer.width / this.device.maxPixelRatio / this.downscale);
```

Comment:
not sure maxPixelRatio should factor into this?

Reply:
Maks added this this in his PR and I think that's correct? If pixel ration says render to half the size screen res .. we should use half the size for internal buffers, unless `colorBuffer.width` already has this scale in?

Reply:
Looking at the code, it seems renderTargetScale is never modified nor publicly exposed / documented variable always holding 1. I suspect we should just remove it.

Reply:
I am guessing this is legacy code from trying to perform dynamic resolution downscaling? Just a speculation though.

Reply:
Using maxPixelRatio here means the SSAO downscale buffer depends on the browser's CSS scaling rules, which I think is wrong.

SSAO downscaled buffer size should be a simple multiple of render target size.